pytmc/bin/db.py

```diff
@@ -196,7 +193,10 @@ def main(tmc_file, record_file=None, *, dbd=None, allow_errors=False,
             tmc, dbd_file=dbd, allow_errors=allow_errors,
             show_error_context=not no_error_context,
         )
-    except LinterError:
+    except LinterError as ex:
```

Comment:
```suggestion
    except LinterError:
```
This is to placate flake8 which takes fault with the unused `ex` variable.